packages/common/src/styles/index.ts

```diff
@@ -1,5 +1,8 @@
+import AppThemeProvider from './ThemeProvider';
 import makeStyles from '@material-ui/core/styles/makeStyles';
 import CssBaseline from '@material-ui/core/CssBaseline';
-import ThemeProvider from './ThemeProvider';
 
-export { CssBaseline, makeStyles, ThemeProvider };
+import { ApplicationTheme } from './theme';
+type AppTheme = ApplicationTheme;
```

Comment:
Without this I keep getting a 'type' not found error but compilation is fine...

Reply:
Not sold on the name, `AppTheme` relates to which app?
I was wondering if we should call it `MSupplyTheme` or similar. This is the global, customised theme, right? which is why the rename?